6_exception1.py

```diff
@@ -4,17 +4,24 @@
 
 Исключения: KeyboardInterrupt
 
-* Перепишите функцию hello_user() из задания while1, чтобы она 
-  перехватывала KeyboardInterrupt, писала пользователю "Пока!" 
+* Перепишите функцию hello_user() из задания while1, чтобы она
+  перехватывала KeyboardInterrupt, писала пользователю "Пока!"
   и завершала работу при помощи оператора break
-    
+
 """
 
 def hello_user():
-    """
-    Замените pass на ваш код
-    """
-    pass
-    
+    try:
+        text = input('Как дела?: ')
```

Comment:
```suggestion
        text = ""
```

логигу не изменит, но можно будет избавиться от повторного вызова input, и лишнего блока try